src/player/tests.rs

```diff
@@ -0,0 +1,52 @@
+use crate::player::{PLAYER_SPEED, Player};
+
+#[test]
+fn stop_moving_stops_the_player() {
+    let mut player = Player::default();
+    player.moving = true;
+
+    player.stop_moving();
+
+    assert_eq!(player.is_moving(), false);
+}
+
+#[test]
+fn move_down_makes_velocity_positive() {
+    let mut player = Player::default();
+    player.move_down();
+
+    assert_eq!(player.is_moving(), true);
+    assert_eq!(player.velocity, PLAYER_SPEED);
+}
+
+#[test]
+fn move_up_makes_velocity_negative() {
+    let mut player = Player::default();
+    player.move_up();
+
+    assert_eq!(player.is_moving(), true);
+    assert_eq!(player.velocity, -PLAYER_SPEED);
+}
+
+#[test]
+fn update_when_player_is_moving() {
```

Comment:
this and the previous could be together

Reply:
Do you mean this one and the **next** one?